enterprise/api/license.go

```diff
@@ -0,0 +1,15 @@
+package api
+
+type License struct {
```

Comment:
also comment for License / LicenseService as exported types.

Reply:
I'll add them, but in my opinion, sometimes some comments are meaningless, they're just repeating their names, like:

```go
// api/plan.go

// PlanType is the type for a plan.
type PlanType int

// api/login.go
// Login is the API message for logins.
type Login struct {}
```

So I used to only comment for unclear variables or complex functions (but these dirty things shouldn't appear, right?)
But it's good to keep the same style under teamwork. So maybe we can add this guideline in our doc?

Reply:
Cause I saw some existed code don't have comments too, so I'm not sure if we should comment everything in api, or just keep necessary comments

Reply:
It's part of Go style. Otherwise golint tool will complain.
  https://go.dev/doc/effective_go#commentary

Almost all languages should have comments (javadoc e.g.) on exported/public stuff, regardless of the richness of the sentence. And we should explain better on the usage of these exports. As we develop the code, we have more context than other folks who are new to the codebase.  We're doing open-source :-). For example, we can add more explanations about what Login does.

Reply:
Quite agree on the open-source part, we shall be more strict to review incoming PRs